lib/src/main/java/com/auth0/spring/security/api/JwtWebSecurityConfigurer.java

```diff
@@ -7,18 +7,27 @@
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.config.http.SessionCreationPolicy;
 
+import java.util.Arrays;
+import java.util.List;
+
 /**
  * Utility class for configuring Security for your Spring API
  */
 public class JwtWebSecurityConfigurer {
 
     final String audience;
-    final String issuer;
+    final List<String> issuers;
     final AuthenticationProvider provider;
 
     private JwtWebSecurityConfigurer(String audience, String issuer, AuthenticationProvider authenticationProvider) {
         this.audience = audience;
-        this.issuer = issuer;
+        this.issuers = Arrays.asList(issuer);
```

Comment:
This constructor is private so there are no breaking changes if we remove it. We should be able to keep only the one accepting multiple issuers.

Reply:
Agreed!